ee/server/service/vpp.go

```diff
@@ -0,0 +1,153 @@
+package service
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/fleetdm/fleet/v4/server/contexts/ctxerr"
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/fleetdm/fleet/v4/server/mdm/apple/itunes"
+	"github.com/fleetdm/fleet/v4/server/mdm/apple/vpp"
+)
+
+func (svc *Service) getVPPToken(ctx context.Context) (string, error) {
+	configMap, err := svc.ds.GetAllMDMConfigAssetsByName(ctx, []fleet.MDMAssetName{fleet.MDMAssetVPPToken})
+	if err != nil {
+		return "", ctxerr.Wrap(ctx, err, "fetching vpp token")
+	}
+
+	var vppTokenData fleet.VPPTokenData
+	if err := json.Unmarshal(configMap[fleet.MDMAssetVPPToken].Value, &vppTokenData); err != nil {
+		return "", ctxerr.Wrap(ctx, err, "unmarshaling VPP token data")
+	}
+
+	return base64.StdEncoding.EncodeToString([]byte(vppTokenData.Token)), nil
```

Comment:
sorry about this, seems like it was a bad idea to store it raw instead of encoded

Reply:
all g! is it worth reversing? could be done in a separate PR

Reply:
yeah, my 2cents: at this point let's add it in the cleanup. IIRC there's another place where we actually need the raw value (which motivated my request) so we should evaluate what happens more often

Reply:
I added a TODO there!